jest_config/__fixtures__/transaction.ts

```diff
@@ -49,6 +49,7 @@ const toTxReceipt = (fixtureTxReceipt: any): ITxReceipt => {
   };
   return {
     ...result,
+    metadata: result.metadata ?? undefined,
```

Comment:
Why the `?? undefined`?

Reply:
Because the fixtures may or may not include it. And we test with strict equality so if they aren't present as undefined it fails.